install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+# -*- sh-basic-offset: 4 -*-
+
+set -euo pipefail
+
+if [[ $(id -u) -ne 0 ]]; then
+    echo "Please run as root or using sudo"
+    exit 1
+fi
+
+mkdir -p /opt/wott
+
+if [ ! -f /etc/wott/config.ini ]; then
+    echo -e "[DEFAULT]\\nenroll_token = $CLAIM_TOKEN" > /opt/wott/config.ini
+fi
+
+# If Ubuntu/Debian is detected
+if [ -f /etc/lsb-release ]; then
```

Comment:
Will not work on Jessie and probably some other flavors. It's better to parse `/etc/os-release` instead, like we do this in Python code.

Reply:
Do we need to check both, or is `/etc/os-release` sufficient for all?

Reply:
`/etc/os-release` is sufficient for all.